ccd-definition/FixedLists/AllocationProposalList.json

```diff
@@ -2,29 +2,29 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "AllocationProposalList",
-    "ListElementCode": "Lay justices",
-    "ListElement": "Magistrate",
+    "ListElementCode": "Circuit judge",
```

Comment:
I think each word in this list should start with a capital letter; our other list of judge titles has every first letter capitalised and I think we should do it here too; can check with a BA as it seems the existing allocation list does not do this (but looks like AC does specify capital letters)